.travis.yml

```diff
@@ -1,20 +1,19 @@
-language: node_js
-node_js:
-  - "6"
-
-sudo: false
+language: generic
+dist: trusty
```

Comment:
That is for the older image. The trusty one dont need it and they are much smaller and faster